_data/projects/ACMESharp.yml

```diff
@@ -1,8 +1,8 @@
+---
 name: ACMESharp
-desc: .NET Library and PowerShell module for the ACME protocol used by Let's Encrypt
+desc: ".NET Library and PowerShell module for the ACME protocol used by Let's Encrypt"
 site: https://github.com/ebekker/ACMESharp
 tags:
-# Note those are tags categorizing your project, not issue labels.
```

Comment:
By round-tripping the YAML we can clean up all these old comments from the template